src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -172,6 +175,25 @@ internal static class DefinitionItemExtensions
             return properties;
         }
 
+        private static ImmutableArray<AdditionalProperty> GetDisplayableProperties(ISymbol definition)
+        {
+            var displayableProperties = ImmutableArray<AdditionalProperty>.Empty;
+
+            var containingType = definition.ContainingType;
+            if (containingType != null)
+            {
+                displayableProperties = displayableProperties.Add(new AdditionalProperty(AbstractReferenceFinder.ContainingTypeInfoPropertyName, containingType.Name));
+            }
+
+            var containingSymbol = definition.ContainingSymbol;
+            if (containingSymbol != null && containingSymbol.GetMemberType() != null)
```

Comment:
why the `.GetMemberType() != null` test?  can you add comment to code explaining the reasoning?